src/Ormolu/Fixity.hs

```diff
@@ -42,12 +42,29 @@ import Data.Set (Set)
 import qualified Data.Set as Set
 import Distribution.Types.PackageName (PackageName, mkPackageName, unPackageName)
 import Ormolu.Fixity.Internal
+#if BUNDLE_FIXITIES
+import Data.FileEmbed (embedFile)
+#else
+import qualified Data.ByteString.Unsafe as BU
+import Foreign.Ptr
+import System.Environment (getEnv)
+import System.IO.Unsafe (unsafePerformIO)
+#endif
 
 packageToOps :: Map PackageName FixityMap
 packageToPopularity :: Map PackageName Int
+#if BUNDLE_FIXITIES
 HackageInfo packageToOps packageToPopularity =
   Binary.runGet Binary.get $
     BL.fromStrict $(embedFile "extract-hackage-info/hackage-info.bin")
+#else
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
Will do :+1: This is due to the WASM backend not yet supporting Template Haskell, see also the commit message of the second commit:

> Should be reverted once the WASM backend supports TH.